<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrugsDbCreator::MoleculeLinkerWidget</class>
 <widget class="QWidget" name="DrugsDbCreator::MoleculeLinkerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="availableDrugsDb"/>
     </item>
     <item>
      <widget class="QComboBox" name="reveiwers">
       <item>
        <property name="text">
         <string>Reviewer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Eric Maeker, MD (FR)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jim Busser, MD (CA)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Graham Chui, MD (NZ)</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeUnreviewed">
       <property name="text">
        <string>Remove unreviewed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Load file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QTableView" name="tableView">
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Search Molecule</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchMol"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Search ATC code</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchATC"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
